configs/_base_/models/deeplabv3plus_r50-d8_vd_contrast.py

```diff
@@ -0,0 +1,54 @@
+# model settings
+norm_cfg = dict(type='SyncBN', requires_grad=True)
+data_preprocessor = dict(
+    type='SegDataPreProcessor',
+    mean=[123.675, 116.28, 103.53],
+    std=[58.395, 57.12, 57.375],
+    bgr_to_rgb=True,
+    pad_val=0,
+    seg_pad_val=255)
+model = dict(
+    type='EncoderDecoder',
+    data_preprocessor=data_preprocessor,
+    pretrained='open-mmlab://resnet50_v1d',
```

Comment:
We do not provide the resnetv1d pre-trained model yet, might set `None` to `pretrained`.

Reply:
`pretrained` has been set `None`, and original code has been annotation.